analyses/oncoprint-landscape/01-plot-oncoprint.R

```diff
@@ -156,18 +162,13 @@ if (!is.null(opt$goi_list)) {
   goi_list <- unique(unlist(goi_list))
 }
 
-# Read in recurrent focal CNVs file
-if (!is.null(opt$focal_file)) {
-  focal_df <- readr::read_tsv(file.path(opt$focal_file))
-}
-
```

Comment:
This is something that should have been removed earlier, correct?

Reply:
Correct!